modules/bibsort/etc/bibsort.cfg

```diff
@@ -17,25 +17,30 @@
 
 [sort_field_1]
 name = latest first
-definition = BIBREC: creation_date
+definition = BIBREC: id
 washer = NOOP
```

Comment:
This change does not seem good.  One of the primary use cases of BibSort was to enable "real" sorting by latest records as first.  Hence we should sort by the time records were created, not by their IDs that may be forced.  E.g. imagine filling new records into record ID hole; we still want to show them as new, depending on their real date.